packages/sdk/cloudflare/example/src/index.ts

```diff
@@ -1,24 +1,30 @@
+/* eslint-disable no-console */
 import { init as initLD } from '@launchdarkly/cloudflare-server-sdk';
 
 export default {
   async fetch(request: Request, env: Bindings): Promise<Response> {
```

Comment:
We'll need to add the context argument to leverage the [`waitUntil` method](https://developers.cloudflare.com/workers/runtime-apis/fetch-event/#waituntil) (more detail below).
```suggestion
  async fetch(request: Request, env: Bindings, context: ExecutionContext): Promise<Response> {
```